app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -400,37 +411,37 @@ msgstr "Byddwch yn cael y cyfle i weld ac argraffu copi o'ch atebion ar ôl cyfl
 msgid "We've received your answers"
 msgstr "Rydym wedi derbyn eich atebion"
 
-#: templates/thank-you.html:8
+#: templates/thank-you.html:7
 msgid "Thank you for submitting your census"
 msgstr "Diolch am gyflwyno eich cyfrifiad"
 
-#: templates/thank-you.html:10
+#: templates/thank-you.html:9
 msgid "We have received your answers and you do not need to do anything else."
 msgstr "Rydym ni wedi cael eich atebion ac nid oes angen i chi wneud unrhyw beth arall."
 
-#: templates/view-submission.html:8
+#: templates/view-submission.html:7
 msgid "Submission"
 msgstr "Cyflwyno"
 
-#: templates/view-submission.html:11
+#: templates/view-submission.html:10
 msgid "Submitted answers"
 msgstr "Atebion a gyflwynwyd"
 
-#: templates/view-submission.html:20
+#: templates/view-submission.html:19
 #, python-format
 msgid "Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_time)s"
 msgstr "Cyflwynwyd eich atebion ar gyfer <span>%(ru_name)s</span> (%(trading_as_name)s) ar %(submitted_time)s"
 
-#: templates/view-submission.html:25
+#: templates/view-submission.html:24
 #, python-format
 msgid "Your answers were submitted for <span>%(ru_name)s</span> on %(submitted_time)s"
 msgstr "Cyflwynwyd eich atebion ar gyfer  <span>%(ru_name)s</span> ar %(submitted_time)s"
```

Comment:
I know these are existing but while you're at it?
```suggestion
msgstr "Cyflwynwyd eich atebion ar gyfer <span>%(ru_name)s</span> ar %(submitted_time)s"
```

Reply:
Assuming the build passes we'll need to leave this as the latest Welsh translations are needed for testing imminently.